src/NetworkTest/types/opentok/index.ts

```diff
@@ -102,7 +101,6 @@ export namespace OT {
   export type SubscriberProperties = OTSubscriber.SubscriberProperties;
   export type TrackStats = OTSubscriber.TrackStats;
   export type OTError = OTError.OTError;
-  export type PublisherRtcStatsReportArr = OTPublisher.PublisherRtcStatsReportArr;
+  export type PublisherStats = OTPublisher.PublisherStats;
```

Comment:
So, is the interface of the results changing? We should rev the major version and document the changes in the readme.